### DISCUSSION TITLE
Magnet and Pickup Upgrades Respect Other Inventory Items

### DISCUSSION THREAD

**Discussion by raz818** [P3pp3rF1y/SophisticatedBackpacks#154] [Feature / Idea Archive] [answered]
When picking up an item while wearing a backpack with either the magnet upgrade or pickup upgrade, other inventory items, such as the Dank Null, are bypassed and the item is placed in the backpack.

In the case of the magnet upgrade, it makes sense that it would bypass another inventory item, as it is the backpack that is pulling in the items, but I wish the pickup upgrade would respect other inventory items.

Would it be possible to add a setting or possibly another upgrade that would allow the prioritization of other inventory items?

**Comment by P3pp3rF1y** [comment]
It's not possible for me to just ignore other items, I can only try to possibly tweak priorities at which these are handled but if dank null happens to handle the event first there's nothing that can be done about that.

**Comment by P3pp3rF1y** [comment, marked as answer]
There isn't anything I can do here. In general defining what is the priority of different items that can be picking up stacks from world would require to have something in forge itself to allow player to do that.
